src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -24,17 +22,25 @@ import angryNature from '../../Assets/angryNature.wav' ;
 import fineNature from '../../Assets/fineNature.wav' ;
 import anxiousNature from '../../Assets/anxiousNature.wav';
 import sadNature from '../../Assets/sadNature.wav' ;
-
+import angryAbstract from '../../Assets/angryAbstract.wav' ;
+import anxiousAbstract from '../../Assets/anxiousAbstract.wav' ;
+import sadAbstract from '../../Assets/sadAbstract.wav' ;
+import fineAbstract from '../../Assets/fineAbstract.wav' ;
+import angryMusic from '../../Assets/angryMusic.mp3'
+import anxiousMusic from '../../Assets/anxiousMusic.mp3'
+import sadMusic from '../../Assets/sadMusic.mp3'
+import fineMusic from '../../Assets/fineMusic.mp3'
```

Comment:
Love the music/sounds! I know we discussed this in person, but I want to note that the user currently can't change music while it is playing. Might be something to reach for in future iterations

Reply:
Just opened a proprity 3 issue on this!